pom.xml

```diff
@@ -1379,12 +1379,12 @@
       <dependency>
         <groupId>org.infinispan.protostream</groupId>
         <artifactId>protostream</artifactId>
-        <version>4.3.5.Final</version>
+        <version>4.4.0.Beta2</version>
       </dependency>
       <dependency>
         <groupId>org.infinispan</groupId>
         <artifactId>infinispan-core</artifactId>
-        <version>11.0.10.Final</version>
+        <version>12.0.1.Final</version>
```

Comment:
The last version is 12.0.2 so it does not make much sense to upgrade to 12.0.1.

Reply:
I have changed the issue and will be looking into upgrading to version 12.0.2.